public/lib/helpers/timeline_helper.js

```diff
@@ -49,16 +49,12 @@ define(function (require) {
      * @returns the label as a string
      */
     TimelineHelper.prototype.pluckLabel = function (hit, params, notify) {
-      let field;
-      if (params.labelFieldSequence) { // in kibi, we have the path property of a field
-        field = kibiUtils.getValuesAtPath(hit._source, params.labelFieldSequence);
-      } else {
-        field = _.get(hit._source, params.labelField);
-      }
-      if (field && (!_.isArray(field) || field.length)) {
-        return field;
-      }
-      return 'N/A';
+      // in kibi, we have the path property of a field
+      const field = params.labelFieldSequence ? params.labelFieldSequence[0] : params.labelField;
+      // get value from hit.fields in case of multi-fields
+      const value = _.get(hit._source, field) ? _.get(hit._source, field) : _.get(hit.fields, field);
```

Comment:
you should use `kibiUtils.getValuesAtPath` instead of `_.get`. The reason is that `kibiUtils.getValuesAtPath` support object in array. For example with the following object:

```json
{
  "aaa": [
    {
      "bbb": 123
    }
  ]
}
```

To retrieve `123` you'd have to to call `_.get` with `[ 'aaa', '0', 'bbb' ]`, while with `kibiUtils.getValuesAtPath` you can just do `[ 'aaa', 'bbb' ]`.

---

With `hit.fields` however, just do `hit.fields[params.labelField]`.